agent/agent.go

```diff
@@ -1807,6 +1810,24 @@ func (a *Agent) WANMembers() []serf.Member {
 	return nil
 }
 
+// IsMember is used to check if a node with the given nodeName
+// is a member
+func (a *Agent) IsMember(nodeName string) bool {
+	for _, m := range a.LANMembers() {
+		if m.Name == nodeName {
+			return true
+		}
+	}
+
+	for _, m := range a.WANMembers() {
```

Comment:
Currently users can only ForceLeave nodes within our DC.

Since WANMembers contains servers in the gossip pool that spans across DCs, this check can be removed until that functionality is supported.

Reply:
ah! will remove the wan stuff then, thanks freddy